doc_source/create-cluster.md

```diff
@@ -89,6 +89,8 @@ If you select subnets that were created before 03/26/2020 using one of the Amazo
 **Important**  
 The worker node AWS CloudFormation template modifies the security group that you specify here, so **Amazon EKS strongly recommends that you use a dedicated security group for each cluster control plane \(one per cluster\)**\. If this security group is shared with other resources, you might block or disrupt connections to those resources\.
    + **Endpoint private access** – Choose whether to enable or disable private access for your cluster's Kubernetes API server endpoint\. If you enable private access, Kubernetes API requests that originate from within your cluster's VPC use the private VPC endpoint\. For more information, see [Amazon EKS cluster endpoint access control](cluster-endpoint.md)\.
+**Important**  
+If you created a VPC with only private subnets, you must enable **Endpoint private access**.
```

Comment:
I don't think this is true

Reply:
A VPC with private subnets should still be allowed to have Endpoint Public Access set IIUC, it's if the VPC has *no internet access*, then the cluster would need to have Endpoint private access enabled.

Reply:
Unless the "only" in "VPC with only private subnets" means there are no NATs that the "private" subnets are associated with.

Reply:
the IGW would technically have to reside in a public subnet, so I think this statement may actually be correct.